### DISCUSSION TITLE
In the case of multiple git projects, how do I determine the repository

### DISCUSSION THREAD

**Discussion by 2564089649** [microsoft/vscode-discussions#1981] [Extension Development QnA] [answered]
I want to develop a feature that:
Add an icon in the scm menu, click icon to get the corresponding git project diff information.
So far I've added icon to the scm menu in the package.josn file.
![image](https://github.com/user-attachments/assets/cd05c1cd-0c1d-4680-9f7d-475b995dc734)
![image](https://github.com/user-attachments/assets/45840ab4-9e24-4d14-bc3c-3ea5b93132a5)
And get the diff of all the repositories.
![image](https://github.com/user-attachments/assets/b089e39b-d8f9-4fe5-9e5e-b07ba0e92680)
However, when there are multiple repositories, how do I determine which repository's icon the user clicked on so that I only get the diff for the corresponding repository?

**Comment by isidorn** [comment]
It is probably passed as the context of your command? Did you check that?

**Reply by 2564089649** [reply, marked as answer]
Thanks for your replay, I just found out that I can do this.
